portal-frontend/src/app/features/notice-of-intents/edit-submission/parcels/parcel-details.component.html

```diff
@@ -0,0 +1,68 @@
+<div>
+  <div class="step-description">
+    <h3>Identify Parcels Under Application</h3>
```

Comment:
I guess application here used more like submission, correct?

Reply:
nah, just forgot to update this, update to Notice of Intent